docs/Parallel-Learning-Guide.rst

```diff
@@ -281,14 +281,16 @@ Saving Dask Models
 
 After training with Dask, you have several options for saving a fitted model.
 
-**Option 1: pickle the Dask estimator**
+**Option 1: Convert to a scikit-learn estimator (Recommended)**
```

Comment:
You're presenting this as a more secure approach to the same functionality, but this is just a fundamental behavior change.

We want to continue support taking a Dask estimator and reloading it as a Dask estimator, with all its Dask-specific details.